### DISCUSSION TITLE
RFC: Improving the search preview

### DISCUSSION THREAD

**Discussion by Strift** [meilisearch/product#766] [Feedback & Feature Proposal]
🔗 Upvote this feature [in our roadmap](https://roadmap.meilisearch.com/c/170-browse-ui?&utm_medium=social&utm_source=starter_share)

## Summary

Improve the [Search Preview](https://www.meilisearch.com/docs/learn/getting_started/search_preview?utm_source=github) application to enable [Meilisearch Cloud](https://meilisearch.com/cloud?utm_source=github) users to test their search experience more easily.

## Motivation

We want to give users an effortless way to test Meilisearch’s features. We realized that new users often struggle to set up their search settings to match their relevancy expectations. Moreover, the engine has features that can’t be tested using the current [open-source mini-dashboard](https://github.com/meilisearch/mini-dashboard). 

## Feedback request

We need your feedback on the [Search Preview](https://www.meilisearch.com/docs/learn/getting_started/search_preview?utm_source=github):
- What’s good and what’s not about the mini-dashboard?
- What do you think could be improved?

Thank your for your time 🙏

**Comment by razzeee** [comment]
Showing facettes would have been useful in some cases. Might also be neat for data exploration where you want to figure out which filters might make sense to offer.

**Reply by macraig** [reply]
Thank you for your feedback @razzeee ! We're planning to add facets and filters to the new preview :)
